File: 0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.java
class Solution {
    public String reverseWords(String s) {
        String[] words=s.split(" ");
        /*String res="";
        for(String w:words){
            String revw=new StringBuilder(w).reverse().toString();
            String uprevw=String.join(" ",revw);
            res+=uprevw;
        }*/
        StringBuilder res=new StringBuilder();
        for(int i=0;i<words.length;i++){
            res.append(new StringBuilder(words[i]).reverse());
            if(i<words.length-1) res.append(" ");
        }
        return res.toString();
    }
}

File: 0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.cpp
class Solution {
public:
    string reverseWords(string s) {
                stringstream ss(s);
        string word,res;
        while(ss>>word){
            reverse(word.begin(),word.end());
            res+=word+" ";
        }
        if(!res.empty()) res.pop_back();
        return res;
    }
};